contrib/raftexample/main.go

```diff
@@ -33,9 +33,14 @@ func main() {
 	confChangeC := make(chan raftpb.ConfChange)
 	defer close(confChangeC)
 
+	commitC, errorC := make(chan *string), make(chan error)
+	walReady := make(chan struct{})
+	kv := newKVStore(proposeC, commitC, errorC, walReady)
```

Comment:
`newKVStore` already synchronously returned when the wal was ready, why add extra channels?